### PR TITLE
Add `version` to message logs

### PR DESCRIPTION
**Pull request by teghnet**

### CODE REVIEW

File: pkg/log/grafana/log.go [hunk 1]
```diff
@@ -515,6 +522,52 @@ func toFloat(value reflect.Value) (float64, bool) {
 	return 0, false
 }
 
+func ToFloatVersion(value reflect.Value) (float64, bool) {
```
Comment (MDobak) [inline pkg/log/grafana/log.go:525]: IMO, this function should't be a part of this pkg. I would see this function as a part of config pkg.

File: pkg/log/grafana/log.go [hunk 2]
```diff
@@ -426,7 +433,7 @@ func match(metric Metric, msg string, fields reflect.Value) bool {
 }
 
 // varRegexp matches vars in format: ${foo}
-var varRegexp = regexp.MustCompile(`\$\{[^}]+\}`)
+var varRegexp = regexp.MustCompile(`\$\{[^}]+}`)
```
Comment (jar-o) [inline pkg/log/grafana/log.go:436]: this

```
var varRegexp := regexp.MustCompile(`\$\{[^}^\s]+}`)
```

would prevent matches on spaces (e.g. `${foo }` which could catch someone out.

https://play.golang.com/p/o9shz78x7UO

Comment (teghnet) [reply pkg/log/grafana/log.go:444]: Indeed. Fixed.

Comment (teghnet) [reply pkg/log/grafana/log.go:444]: I do not want to be too permissive. If we have a space in the name, then we expect a name with the space.
I think.

File: pkg/log/grafana/log.go (hunk 1 above)
Comment (teghnet) [reply pkg/log/grafana/log.go]: moved

**Comment by teghnet** [comment]
Closed due to being included in #68

**Pull request closed without merging**